aws/structure.go

```diff
@@ -3724,6 +3725,47 @@ func flattenWafWebAclRules(ts []*waf.ActivatedRule) []map[string]interface{} {
 	return out
 }
 
+func flattenWorkLinkNetworkConfigResponse(c *worklink.DescribeCompanyNetworkConfigurationOutput) []map[string]interface{} {
+	config := make(map[string]interface{})
+
+	if c == nil {
+		return nil
+	}
+
+	var emptyVpc bool
+	if c.VpcId == nil || *c.VpcId == "" {
+		emptyVpc = true
+	}
+	if len(c.SubnetIds) == 0 && len(c.SecurityGroupIds) == 0 && emptyVpc {
```

Comment:
Nit: `emptyVpc` can be removed in favor of just `&& aws.StringValue(c.VpcId) == ""`

Reply:
Fixed